core/nginx/conf/nginx.conf

```
@@ -95,18 +95,18 @@ http {
       client_max_body_size {{ MESSAGE_SIZE_LIMIT|int + 8388608 }};
 
       # Listen on HTTP only in kubernetes or behind reverse proxy
-      {% if KUBERNETES_INGRESS or TLS_FLAVOR in [ 'mail-letsencrypt', 'notls', 'mail' ] %}
-      listen 80{% if PROXY_PROTOCOL in ['all', 'http'] %} proxy_protocol{% endif %};
+      {% if TLS_FLAVOR in [ 'mail-letsencrypt', 'notls', 'mail' ] %}
+      listen 80{% if PROXY_HTTPPROTOCOL_80 %} proxy_protocol{% endif %};
```

This is probably a typo and should be `PROXY_PROTOCOL_80`

```suggestion
      listen 80{% if PROXY_PROTOCOL_80 %} proxy_protocol{% endif %};
```